packages/react-native-editor/__device-tests__/helpers/utils.js

```diff
@@ -464,11 +464,27 @@ const waitForMediaLibrary = async ( driver ) => {
 	await waitForVisible( driver, locator );
 };
 
-const waitForVisible = async ( driver, elementLocator, iteration = 0 ) => {
-	const maxIteration = 25;
+/**
+ *
+ * @param {string} driver
+ * @param {string} elementLocator
+ * @param {boolean} returnBool - To return boolean instead of the first element/throw error. Useful for asserts.
+ * @param {number} iteration - Default value is 0
+ */
+const waitForVisible = async (
+	driver,
+	elementLocator,
+	returnBool,
+	iteration = 0
+) => {
+	const maxIteration = returnBool ? 5 : 25;
```

Comment:
Ah right, sorry it's not clear, in the one case that relied on the bool value, it was waiting for the element to not be visible instead of waiting for it to be visible. 5 was a magic number since I thought it didn't need to wait that long to validate that. 

It probably doesn't need to wait 25 seconds for that, I think 5 seconds is enough, what do you think?

Reply:
Oh ok, that makes sense. In that case, we might consider setting the max iteration value via an argument, this could be helpful for example if `isElementVisible` function uses this helper and wants to use a lower max iteration to reduce the waiting.

Reply:
That's a good point, it would make it more flexible. I've updated it in this commit: https://github.com/WordPress/gutenberg/pull/40670/commits/85c407c286fc18e226f98fbcfa440237f995d093